instruments/config.py

```diff
@@ -12,9 +12,22 @@
 import warnings
 
 try:
-    import yaml
+    import ruamel.yaml as yaml
 except ImportError:
-    yaml = None
+    # Some versions of ruamel.yaml are named ruamel_yaml, so try that
+    # too.
```

Comment:
Anaconda, at least, seems to rename it to avoid a conflict somewhere. Ran into that trying to locally reproduce the pylint failure.